vue.config.js

```diff
@@ -39,4 +39,21 @@ module.exports = {
       },
     },
   },
+
+  devServer: {
+    proxy: {
+      '/game': {
+        target: 'http://localhost:1337',
+        changeOrigin: true,
+      },
+      '/user': {
+        target: 'http://localhost:1337',
+        changeOrigin: true,
+      },
+      '/test': {
+        target: 'http://localhost:1337',
+        changeOrigin: true,
+      },
+    },
+  },
```

Comment:
Needed for the client calls to get data from the server successfully and set the cookie.

Reply:
Does this not affect prod/mockprod?